bin/tictactoe

```diff
@@ -1,3 +1,43 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
```

Comment:
For lines 5-43. I would abstract this into a CLI class in the lib folder and give them methods you can call. 

On line 5 we could just do this now

```ruby 
CLI.new.start_game
```